.github/workflows/CI.yml

```
@@ -34,21 +34,24 @@ on:
 
 jobs:
   UTTEST:
+    strategy:
+      fail-fast: false
```

Why we disable the fail fast?